### PR TITLE
COMMERCE-12244 | Discount Handling of UOM (part 1)

### PR DESCRIPTION
**Pull request by ghost**
https://liferay.atlassian.net/browse/COMMERCE-12244

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/frontend/data/set/view/table/CommerceDiscountCPInstanceTableFDSView.java [hunk 1]
```diff
@@ -37,6 +37,8 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 				"actionLink")
 		).add(
 			"productName.LANG", "product"
+		).add(
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-pricing-web/src/main/java/com/liferay/commerce/pricing/web/internal/frontend/data/set/view/table/CommerceDiscountCPInstanceTableFDSView.java:40]: This should go under feature flag

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
Please add feature flag management.

Thank you!

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:stop

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.

Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Review by luca-pellizzon — APPROVED** [review]

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
ci:test:commerce-functional

**Comment by CrescenzoRegaLF** [comment]
sent here
https://github.com/liferay-commerce/liferay-portal/pull/4243

**Pull request closed without merging**